prior_localization/prepare_data.py

```diff
@@ -272,17 +284,16 @@ def prepare_behavior(
             set_neurometrics = []
             for i in range(len(pseudo_ids)):
                 neurometrics = compute_neurometric_prior(all_trials[i], session_id, subject, model, behavior_path)
-                set_neurometrics.append(neurometrics[trials_mask].reset_index(drop=True))
+                set_neurometrics.append(neurometrics[trials_mask].reset_index(drop=True))  # TODO: need to update mask
```

Comment:
Just seeing the TODO -- what's meant here? Is that something that needs to change? Seems like it could easily be forgotten, so if it's essential can we do it now? Or is it not essential?